Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rust:1.43.1 as build
```

Comment:
Can we maybe get an alipne (=more lightweight) version? or is this one the lightest one

Reply:
This is a multi stage build, and is only used for compilation.